Task_2/Feeder.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_2
+{
+    class Feeder
+    {
+        Random rand = new Random();
```

Comment:
Since it is a filed, it should be a private.
Also initialization should be performed in a constructor.